### ISSUE TITLE
Filter icons by latest, name, etc

### ISSUE THREAD

**Issue by RyanRoberts** [elrumo/macOS_Big_Sur_icons_replacements#525]
Just a feature request for the site, any chance of a filter to order the icons by latest and/or other filters like list alphabetically? 

I often hit the site to see what's new.

Great work on this btw!

**Comment by elrumo** [comment]
Good idea, yesterday I had some time and started working on that. But unfortunately I can't think of an easy way to make that possible at the moment as we don't have the timestamp for when the files were added on the icns.txt file, so I've taken this to address another other problem we had: people who are not familiar with GitHub could not contribute. I've started making a submission system where anyone will be able to upload new icons and a more traditional database will be used to store other data like timestamps. 

In the meantime, this is what the filter option button would look like:
![image](https://user-images.githubusercontent.com/36299870/94918128-d9809980-04a9-11eb-8676-0fe742f0587e.png)
